src/seedu/addressbook/commands/SortCommand.java

```diff
@@ -0,0 +1,17 @@
+package seedu.addressbook.commands;
+import seedu.addressbook.data.person.ReadOnlyPerson;
+import java.util.List;
+/**
+ * Sorts all persons in the address book to the user.
+ */
+public class SortCommand extends Command{
+    public static final String COMMAND_WORD = "sort";
```

Comment:
Very good!  Constant names written using all uppercase using underscore to separate words.